### PR TITLE
Playground: Fix performance issues

### PR DESCRIPTION
**Pull request by schlenkibus**

### CODE REVIEW

File: playground/src/proxies/lpc/LPCProxy.h [hunk 1]
```diff
@@ -103,6 +104,7 @@ class LPCProxy
 
   int m_lpcSoftwareVersion = 0;
   Signal<void, int> m_signalLPCSoftwareVersionChanged;
+  std::shared_ptr<DebugLevel> m_debugSetting;
```
Comment (hhoegelo) [inline playground/src/proxies/lpc/LPCProxy.h]: I don't think this is useful. In fact, I would like to make the settings uniquely owned by the Settings-object, so spreading its ownership all over the app is the wrong direction.

File: playground/src/serialization/ParameterGroupsSerializer.cpp [hunk 2]
```diff
@@ -29,6 +30,11 @@ void ParameterGroupsSerializer::readTagContent(Reader &reader) const
 {
   reader.onTag(ParameterGroupSerializer::getTagName(), [&](const Attributes &attr) mutable {
     auto group = m_paramGroups->getParameterGroupByID(attr.get("id"));
-    return new ParameterGroupSerializer(group);
+    if(group) {
+        return new ParameterGroupSerializer(group);
+    } else {
+        DebugLevel::error("could not parse parameter group");
```
Comment (hhoegelo) [inline playground/src/serialization/ParameterGroupsSerializer.cpp]: Did this happen to you? If so, we should probably know the reason?

File: playground/src/presets/EditBuffer.h [hunk 3]
```diff
@@ -105,7 +105,9 @@ class EditBuffer : public ParameterGroupSet
 
   Uuid m_lastLoadedPreset;
   tUpdateID m_updateIdWhenLastLoadedPresetChanged = 0;
-  Glib::ustring m_name;
+
+private:
```
Comment (hhoegelo) [inline playground/src/presets/EditBuffer.h]: the private tag in the private section is redundant

File: playground/src/parameters/Parameter.h [hunk 4]
```diff
@@ -139,6 +139,7 @@ class Parameter : public UpdateDocumentContributor,
   Signal<void, const Parameter *> m_signalParamChanged;
 
   uint16_t m_id;
+  mutable PresetParameter* m_cachedOGParam;
```
Comment (hhoegelo) [inline playground/src/parameters/Parameter.h]: I am not happy with the m_cachedOGParam. Can we be sure, that the object lives as long as the edit buffer - or is the pointer maintained by the parameter pointed to?

File: playground/src/serialization/ParameterGroupsSerializer.cpp (hunk 2 above)
Comment (schlenkibus) [reply playground/src/serialization/ParameterGroupsSerializer.cpp]: It was premature, I still had my Layer-Foo editbuffer in my preset-manager. I will remove that.

File: playground/src/parameters/Parameter.h (hunk 4 above)
Comment (schlenkibus) [reply playground/src/parameters/Parameter.h]: I will review that, also Stephan found a bug which I can pinpoint to not being cautious enough with this pointer. I will see what implementation will be less error prone and easier to maintain

**Pull request closed without merging**